cypress/integration/james.spec.js

```diff
@@ -0,0 +1,23 @@
+context('Actions', () => {
+    beforeEach(() => {
+      cy.visit('https://google.com')
+    })
+
+    // find the search bar
+    it('should find the input and click on it', () => {
+    cy.get('.gLFyf').click();
+    });
+    // enter text into search bar
+    it('should enter text into search bar', () => {
+        cy.get('.gLFyf').type('planning center online');
+        cy.get('.gLFyf').type('{enter}');
```

Comment:
You can combine these two lines like this:
`cy.get('.gLFyf').type('planning center online{enter}');`